File: cases/taint/javascript/array-element-negative/ArrayElementNegative.js
function dfb_source() { // DFB-SOURCE: array-element-input
  return "tainted";
}

function dfb_sink(value) {} // DFB-SINK: array-element-sink

function run() {
  const values = ["clean", "clean"];
  values[0] = dfb_source(); // DFB-WITNESS: array-element-store
  values[1] = "clean";
  dfb_sink(values[1]);
}

File: cases/taint/cpp/array-element-negative/array_element_negative.cpp
int dfb_source() { // DFB-SOURCE: array-element-input
    return 1;
}

void dfb_sink(int value) {} // DFB-SINK: array-element-sink

void run() {
    int values[2];
    values[0] = dfb_source(); // DFB-WITNESS: array-element-store
    values[1] = 0;
    dfb_sink(values[1]);
}
